service/src/main/java/bio/terra/pipelines/dependencies/stairway/JobService.java

```diff
@@ -103,12 +111,12 @@ public void initialize() {
             .dataSource(stairwayDatabaseConfiguration.getDataSource())
             .context(flightBeanBag)
             .addHook(mdcHook)
-            .addHook(new MonitoringHook())
+            .addHook(new MonitoringHook(openTelemetry))
             .exceptionSerializer(new StairwayExceptionSerializer(objectMapper)));
   }
 
   /** Retrieves Job Result specifying the result class type. */
-  @Traced
+  @WithSpan
```

Comment:
are there any docs we can link to like in the readme to explain this tracing stuff?

Reply:
probably Doug's blog post (which is linked in the tracing ticket), or stuff that's linked in it. do you want me to add that documentation in this PR?